homeassistant/components/meteoclimatic/translations/ca.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
New languages should not be added here. Translations can be managed via Lokalised after this PR is merged.